lnrpc/invoicesrpc/invoices.proto

```diff
@@ -135,3 +141,11 @@ message SubscribeSingleInvoiceRequest {
     // Hash corresponding to the (hold) invoice to subscribe to.
     bytes r_hash = 2;
 }
+
+message LookupInvoiceMsg {
```

Comment:
Maybe mark the main rpc server `LookupInvoice` as deprecated right away?

Reply:
Sure makes sense. 

I had also started to extend `SettlInvoice` to also take a `set_id` to implement analogous hold logic as keysend, but the scope started to grow and I just wanted to get this smaller bit out the door.

Reply:
Yes, what you have now is indeed a great intermediate step that adds a lot of value.